### DISCUSSION TITLE
gatsby-source-wordpress stops working

### DISCUSSION THREAD

**Discussion by tonyvu2014** [gatsbyjs/gatsby#35922] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I used  `gatsby-source-wordpress` to fetch data from Wordpress and create blog posts. It has been working and then suddenly stopped working one day which failed my gatsby build step.

This is the error:

> Step #4:  gatsby-source-wordpress  Error: socket hang up
> 
> Step #4:     at connResetException (internal/errors.js:628:14)
> 
> Step #4:     at TLSSocket.socketOnEnd (_http_client.js:499:23)
> 
> Step #4:     at TLSSocket.emit (events.js:387:35)
> 
> Step #4:     at endReadableNT (internal/streams/readable.js:1317:12)
> 
> Step #4:     at processTicksAndRejections (internal/process/task_queues.js:82:21)

package version: 
- node: v14
- gatsby: ^4.9.0
- gatsby-source-wordpress: ^6.9.0

What I have tried without success
- Increased the wordpress server memory limit
-  Changed the plugin  schema to: 
`schema: {
          queryDepth: 5,
          perPage: 10,
          timeout: 1000000
}`

Please help

### Reproduction Link

https://github.com/tonyvu2014/compare-n-save

### Steps to Reproduce

1. Trigger the gatsby project build
2.
3.
...

### Expected Result

It should successfully fetch data from Wordpress and build blog post pages

### Actual Result

The build failed due to communication issue with Wordpress server

### Environment

```shell
System:
    OS: macOS 12.2.1
    CPU: (10) x64 Apple M1 Pro
    Shell: 5.8 - /bin/zsh
Binaries:
    Node: 14.19.3 - ~/.nvm/versions/node/v14.19.3/bin/node
    npm: 6.14.17 - ~/.nvm/versions/node/v14.19.3/bin/npm
Languages:
    Python: 2.7.18 - /usr/bin/python
Browsers:
    Chrome: 102.0.5005.61
    Safari: 15.3
npmPackages:
    gatsby: ^4.16.0 => 4.16.0
    gatsby-plugin-image: ^2.16.1 => 2.16.1
    gatsby-plugin-sharp: ^4.16.1 => 4.16.1
    gatsby-source-filesystem: ^4.16.0 => 4.16.0
    gatsby-transformer-sharp: ^4.16.0 => 4.16.0
```

### Config Flags

N/A

**Comment by LekoArts** [comment]
This is a network issue with your WordPress instance, you'll want to reach out to your provider to see if you're hitting their API too hard.
